cardano-cli/cardano-cli.cabal

```diff
@@ -33,8 +38,6 @@ common project-config
                         -Wpartial-fields
                         -Wredundant-constraints
 
-  if impl(ghc >= 9.6)
-    ghc-options:        -Wunused-packages
```

Comment:
Need to remove this to avoid warnings.

Was not able to conditionally disable the warning.